### PR TITLE
Add WPS to adopters

### PR DESCRIPTION
**Pull request by so2bin**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Add WPS to adopters

**Special notes for your reviewer**:

### CODE REVIEW

File: src/pages/adopters.md [hunk 1]
```diff
@@ -45,4 +45,5 @@ All organisations are sorted alphabetically by the first letter of their English
...
+| ZTO                       | https://www.zto.com           | Use Karmada to build multi-cluster platform                                                                                                                                                                                |                                                                                                                                                                          |
+| WPS                       | https://www.wps.cn           | Use Karmada to build a multi-cluster and multi-cloud AI platform.                                                                                                                                                                                 |                                                                                                                                                                          |
```
Comment (windsonsea) [inline src/pages/adopters.md:49]:
```suggestion
| WPS                       | https://www.wps.cn           | Use Karmada to build a multi-cluster and multi-cloud AI platform                                                                                                                                                                                 |                                                                                                                                                                          |
```
For consistency

**Review by windsonsea — COMMENTED** [review]
nit

File: src/pages/adopters.md (hunk 1 above)
Comment (RainbowMango) [reply src/pages/adopters.md:49]: no big deal. I think we can have the `.` at the end.
Sure, we can make an update for all of them. Thanks for pointing that out!

**Comment by RainbowMango** [comment]
@so2bin Thank you! We are glad to have you join the community!

/lgtm
/approve

**Pull request merged**